### PR TITLE
[DEVX-1076] Document sharding for playwright

### PR DESCRIPTION
**Pull request by alexplischke**
Document sharding for playwright.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 1]
```diff
@@ -407,6 +407,20 @@ One or more paths to the playwright test files to run for this suite. Regex valu
 ```yaml
     testMatch: ["**/*.js"]
 ```
+---
+
+#### `numShards`
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/playwright.md:412]:
```suggestion
### `numShards`
```
I think you mean for this to be at the same level as the other `suite.x` properties, not a sub of `suite.testMatch`?

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 2]
```diff
@@ -407,6 +407,20 @@ One or more paths to the playwright test files to run for this suite. Regex valu
...
+<p><small>| OPTIONAL | INTEGER | <span class="highlight playwright">Playwright version >= 1.12</span> |</small></p>
+
+Sets the number of separate shards to create for the test suite. Read more about shard tests on the [Playwright developer site](https://playwright.dev/docs/test-parallel#shards).
+
+When sharding is configured, `saucectl` automatically creates the sharded jobs based on the number of shards you specify. For example, for a suite that specifies 2 shards, `saucectl` clones the suite and runs shard `1/2` on the first suite, and the other shard `2/2` on the identical clone suite.
+
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/playwright.md:418]: Do we need to include the note about uneven test distribution across shards, or is that specific to the way Espresso handles sharding? See the note (just merged yesterday) here: https://docs.saucelabs.com/testrunner-toolkit/configuration/espresso#testoptions.

**Review by sweeneyskirt-sl — COMMENTED** [review]
Just a couple clarifying notes.

File: docs/testrunner-toolkit/configuration/playwright.md (hunk 2 above)
Comment (alexplischke) [reply docs/testrunner-toolkit/configuration/playwright.md:418]: Yeah I copied this section from espresso and removed the notes about uneven handling on purpose :D
To know whether this issue plagues playwright, one would have to inspect the playwright source code on how they split the tests. I didn't notice any empty jobs during my test runs, so we'll have to see 🤞

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**